.github/workflows/ci.yml

```diff
@@ -33,10 +33,11 @@ jobs:
       matrix:
         target: [
           { id: 'linux-amd64', os: 'ubuntu-latest' },
+          { id: 'linux-aarch64', os: ['self-hosted', 'aarch64'] },
```

Comment:
The OS is wrong.
It has to be:

```
            os: [self-hosted, linux, ARM64]
```

As in the Wasmer workflow:
https://github.com/wasmerio/wasmer/blob/master/.github/workflows/main.yaml#L70